src/main.ts

```diff
@@ -20,7 +20,6 @@ export default class FileHider extends Plugin {
 
 	async onload() {
 		await this.loadSettings();
-
```

Comment:
I liked this line being here because it helped separate the settings loading and the event registration. If they were related to each other I'd be fine without the line here, but since they are two separate concepts/things, I want this line to stay here preferably